### PR TITLE
Homogenise notebooks

### PR DESCRIPTION
**Pull request by chrieke**
Homogenizes data block example notebooks, see commit messages.

Items:
* [ ] Ran test & live-tests
* [ ] Implemented (new) unit tests
* [ ] Removed credentials
* [ ] Removed argument pointing to staging
* [ ] Updated [documentation](sdk.up42.com)

For release:
* [ ] Bumped version
* [ ] Added changelog
* [ ] Updated announcement banner

### CODE REVIEW

File: examples/data-block-examples/hexagon-hxgn-content-30cm-download-example.ipynb [hunk 1]
```diff
@@ -1,30 +1,14 @@
...
-   "cell_type": "code",
-   "execution_count": 2,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "# link to the block details page: https://marketplace.up42.com/block/a0766df2-6cdd-42d8-b74a-3ba160dcdb43"
+    "# HxGN Content Program, 30cm (Download) Block\n",
+    "\n",
+    "This block provides RGB+NIR imagery with 30 cm GSD in GeoTIFF across the entire US, Europe and parts of Canada.\n",
```
Comment (markusuwe) [inline examples/data-block-examples/hexagon-hxgn-content-30cm-download-example.ipynb:9]:
```suggestion
    "This block provides RGB+NIR imagery with 30 cm GSD in GeoTIFF format across the entire US, Europe and parts of Canada.\n",
```

File: examples/data-block-examples/nextmapone-viewshed-data-example.ipynb [hunk 2]
```diff
@@ -71,52 +71,20 @@
...
-    "            -105.29594,\n",
-    "            40.049061\n",
-    "          ]\n",
-    "          ]\n",
-    "        ]\n",
-    "      }\n",
-    "    }\n",
-    "  ]\n",
-    "}"
+    "nexamp_example_aoi = {\"type\": \"FeatureCollection\",\"features\": [{\"type\": \"Feature\",\"properties\": {},\n",
```
Comment (markusuwe) [inline examples/data-block-examples/nextmapone-viewshed-data-example.ipynb:77]:
```suggestion
    "nextmap_example_aoi = {\"type\": \"FeatureCollection\",\"features\": [{\"type\": \"Feature\",\"properties\": {},\n",
```

**Review by markusuwe — APPROVED** [review]
LGTM

File: examples/data-block-examples/osm-example.ipynb [hunk 3]
```diff
@@ -6,9 +6,9 @@
    "source": [
     "# Open Street Map Example\n",
     "\n",
-    "This example shows how to use the [**Open Street Map**](https://marketplace.up42.com/block/df2ec03a-50c4-47ac-8a83-2db613869cf9) data block with the UP42 Python SDK. This block enables use of the OSM vector layers (e.g. streets, water bodies, building footprints etc) with UP42. See the [block documentation](https://docs.up42.com/up42-blocks/data/openstreetmap.html?_ga=2.99459805.254624134.1615213941-658905462.1602976684) for more details.\n",
+    "This example shows how to use the Open Street Map data block with the UP42 Python SDK. This block enables use of the OSM vector layers (e.g. streets, water bodies, building footprints etc) with UP42. In this example, we use the UP42 SDK and geopandas to query and visualize mapped [public bookcases](https://en.wikipedia.org/wiki/Public_bookcase) in Brooklyn, New York City. \n",
```
[the remark this answers is not in the document]
Comment (chrieke) [reply examples/data-block-examples/osm-example.ipynb:9]: All resolved, thanks!

**Pull request merged**